### PR TITLE
CMS124 R4 Patient Generation.

### PR DESCRIPTION
**Pull request by hossenlopp**
- Replaces STU3 data with fresh data with a measure report.
- Removes `.cql` file used to STU3 and replaces it with using bundles from Connectathon repo.
- Adds R4 generation.
- Adds R4 patients.

For testing R4 generation, modify line 71 of `Makefile` to use `exm_124_r4` branch of Synthea if https://github.com/projecttacoma/synthea/pull/11 is not merged yet.

### CODE REVIEW

File: EXM_124/Makefile [hunk 1]
```diff
@@ -1,63 +1,98 @@
...
+	curl -X PUT \
+  http://localhost:8080/cqf-ruler-dstu3/fhir/Library/library-EXM124-FHIR3-7.2.000 \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/fhir3/bundles/EXM124_FHIR3-7.2.000/EXM124_FHIR3-7.2.000-files/library-EXM124_FHIR3-7.2.000.json
 	touch .seed-cqf-ruler-no-vs
 
+.seed-cqf-ruler-no-vs-r4:
+	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-r4`; do printf '.'; sleep 5; done
+	curl -X PUT \
+  http://localhost:8080/cqf-ruler-r4/fhir/Measure/measure-EXM124_FHIR4-8.2.000 \
```
Comment (lukoktonos) [inline EXM_124/Makefile]:
```suggestion
  http://localhost:8080/cqf-ruler-r4/fhir/Measure/measure-EXM124-FHIR4-8.2.000 \
```

**Review by lukoktonos — CHANGES_REQUESTED** [review]
- [x] stu3 works
- [x] stu3 no-vs works 
- [x] r4 works
- [x] r4 no-vs works (~_ to -, suggested fix~ fixed)

Just 1 small fix then looks good

**Review by lukoktonos — APPROVED** [review]
Looks good!

**Review by mgramigna — APPROVED** [review]
Looks good

**Pull request merged**